spec/factories/reservations.rb

```diff
@@ -5,6 +5,6 @@
     total { rand(1..300) }
     reserved_date { DateTime.now }
     city { FFaker::Address.city }
-    user_id { nil } # TODO: Modify to user when user is created
+    user { nil }
```

Comment:
Same comment as before for the association.

Reply:
I didn't know we had it this way. Definitely, it suits the test cases much better. I'm going to change it right away. Thanks for the explanation!